src/sig_stfl/xmss/LICENSE

```diff
@@ -0,0 +1,12 @@
+## License
+
+This XMSS reference implementation is Copyright (c) 2024 SandboxAQ and licensed under both the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt) and [MIT License](LICENSE-MIT).
+
+Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any additional terms or conditions.
+
+This XMSS reference implementation is based on the [XMSS reference implementation written by Andreas Hülsing and Joost Rijneveld](https://github.com/XMSS/xmss-reference#license) provided under the CC0 1.0 Universal Public Domain Dedication.
```

Comment:
This looks like a license change from CC0 to Apache2|MIT, no? Do the original authors concur with this?

Reply:
Hi @baentsch , I replied to this point in https://github.com/open-quantum-safe/liboqs/pull/1650#discussion_r1471775460

Reply:
solved in https://github.com/open-quantum-safe/liboqs/pull/1650/commits/59703957f4ee042f01e833243417f605e902800d. Add CC0-1.0 to LICENSE.